config-templates/module_perun.php

```diff
@@ -25,6 +25,8 @@
 	 */
 	'ldap.hostname' => 'ldaps://perun.inside.cz',
 
+	'ldap.base' => 'dc=perun,dc=cesnet,dc=cz',
```

Comment:
Change it to dc=perun,dc=inside,dc=cz to be consistent with the ldap.hostname example